server/common/oursrc/scripts-proxy/main.go

```diff
@@ -60,6 +60,7 @@ func (l *ldapTarget) HandleConn(netConn net.Conn) {
 			log.Printf("resolving default pool: %v", err)
 		}
 	}
+	// TODO: Serve an error page? Forward to scripts-director?
```

Comment:
Sorry server on the director seems reasonable.

Reply:
I suspect I didn't just implement that because I didn't want to think through what that would entail from an IP routing perspective. I'll leave it as a TODO for later.

Reply:
Would it entail anything? If you rewrite the dst addr to be that of a director IP it will Just Work.

Reply:
Uh, so now we're spoofing a client IP on a connection to a director. The director would need to send replies for that traffic back to the proxy so it could send them back to the client and I really don't want to think about making sure all the policy routing is set up correctly in both places, at least as part of this PR.

Reply:
Fair enough. In that case can we serve a simple 500 directly?

Reply:
Yeah. I left the TODO, however, since I agree sorry server would be better.